README.md

```diff
@@ -107,32 +174,31 @@ import marqo
 mq = marqo.Client(url='http://localhost:8882')
 
 mq.create_index("my-first-index")
-mq.index("my-first-index").add_documents(
-    [
-        {
-            "Title": "The Travels of Marco Polo",
-            "Description": "A 13th-century travelogue describing Polo's travels"},
-        {
-            "Title": "Extravehicular Mobility Unit (EMU)",
-            "Description": "The EMU is a spacesuit that provides environmental protection, "
-                           "mobility, life support, and communications for astronauts",
-            "_id": "article_591"
-        }
-    ], 
-    tensor_fields=["Title", "Description"]
```

Comment:
What is the reason we remove `"Title"` from the tensor_fields?

Reply:
Seems like the unstructured indexes perform well with only one tensor field, and since this README is the front page of Marqo, we want it to perform well with results